megamek/src/megamek/common/Board.java

```diff
@@ -1093,10 +1075,10 @@ public void load(InputStream is, StringBuffer errBuff, boolean continueLoadOnErr
         if (isValid(nd, nw, nh, errBuff) && ((nw > 1) || (nh > 1) || (di == (nw * nh)))) {
             newData(nw, nh, nd, errBuff);
         } else if (continueLoadOnError && ((nw > 1) || (nh > 1) || (di == (nw * nh)))) {
-            System.err.println("Error reading board, data invalid.");
+            MegaMek.getLogger().error(this, "Invalid board data!");
             newData(nw, nh, nd, errBuff);
         } else if (errBuff == null){
-            System.err.println("Error reading board, data invalid.");
+            MegaMek.getLogger().error(this, "Invalid board data!");
```

Comment:
Same error message twice in the same method without a way to tell them apart, which could cause issues in the future.